contracts/payloads/IGP106/PayloadIGP106.sol

```diff
@@ -0,0 +1,566 @@
+        {
+            FluidLiquidityAdminStructs.RateDataV2Params[]
+                memory params_ = new FluidLiquidityAdminStructs.RateDataV2Params[](
+                    3
+                );
+
+            // USDC rate curve
+            params_[0] = FluidLiquidityAdminStructs.RateDataV2Params({
+                token: USDC_ADDRESS, // USDC
+                kink1: 85 * 1e2, // 85%
+                kink2: 93 * 1e2, // 93%
+                rateAtUtilizationZero: 0, // 0%
+                rateAtUtilizationKink1: 7 * 1e2, // 7%
+                rateAtUtilizationKink2: 9 * 1e2, // 9%
+                rateAtUtilizationMax: 25 * 1e2 // 25%
+            });
+
+            // USDT rate curve
+            params_[1] = FluidLiquidityAdminStructs.RateDataV2Params({
+                token: USDT_ADDRESS, // USDT
+                kink1: 85 * 1e2, // 85%
+                kink2: 93 * 1e2, // 93%
+                rateAtUtilizationZero: 0, // 0%
+                rateAtUtilizationKink1: 7 * 1e2, // 7%
+                rateAtUtilizationKink2: 9 * 1e2, // 9%
+                rateAtUtilizationMax: 15 * 1e2 // 15%
+            });
+
+            // GHO rate curve
+            params_[2] = FluidLiquidityAdminStructs.RateDataV2Params({
+                token: GHO_ADDRESS, // GHO
+                kink1: 85 * 1e2, // 85%
+                kink2: 93 * 1e2, // 93%
+                rateAtUtilizationZero: 0, // 0%
+                rateAtUtilizationKink1: 7 * 1e2, // 7%
+                rateAtUtilizationKink2: 9 * 1e2, // 9%
+                rateAtUtilizationMax: 15 * 1e2 // 15%
+            });
+
+            LIQUIDITY.updateRateDataV2s(params_);
+        }
+    }
+
+    // @notice Action 3: Update wstUSR stable T1 vault parameters
+    function action3() internal isActionSkippable(3) {
+        uint256 CF = 90 * 1e2;
+        uint256 LT = 92 * 1e2;
+
+        // Update wstUSR T1 vault parameters
+        {
+            address wstUSR_USDC_VAULT = getVaultAddress(110);
+            
+            IFluidVaultT1(wstUSR_USDC_VAULT).updateCollateralFactor(CF);
+            IFluidVaultT1(wstUSR_USDC_VAULT).updateLiquidationThreshold(LT);
+        }
+
+        {
+            address wstUSR_USDT_VAULT = getVaultAddress(111);
+            IFluidVaultT1(wstUSR_USDT_VAULT).updateCollateralFactor(CF);
+            IFluidVaultT1(wstUSR_USDT_VAULT).updateLiquidationThreshold(LT);
+        }
+
+        {
+            address wstUSR_GHO_VAULT = getVaultAddress(112);
+            IFluidVaultT1(wstUSR_GHO_VAULT).updateCollateralFactor(CF);
+            IFluidVaultT1(wstUSR_GHO_VAULT).updateLiquidationThreshold(LT);
+        }
+    }
+
+    // @notice Action 4: Set wstUSR smart vaults T3 and WSTUSR / USDTB vault launch limits
+    function action4() internal isActionSkippable(4) {
+        {
+            // dust limits for wstUSR/USDTb vault
+            address wstUSR_USDTb_VAULT = getVaultAddress(142);
+
+            // [TYPE 1] WSTUSR/USDTbvault - Dust limits
```

Comment:
```suggestion
            // [TYPE 1] WSTUSR/USDTbvault - launch limits
```

we can also just remove some of these comments, it's mentioned many times :laughing: